src/main/java/pl/project13/maven/git/GitMojo.java

```diff
@@ -0,0 +1,326 @@
+package pl.project13.maven.git;
+
+import org.apache.maven.execution.MavenSession;
+import org.apache.maven.plugin.AbstractMojo;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugin.MojoFailureException;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.apache.maven.project.MavenProject;
+import org.jetbrains.annotations.NotNull;
+import pl.project13.maven.git.log.LoggerBridge;
+import pl.project13.maven.git.log.MavenLoggerBridge;
+
+import java.io.File;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.text.SimpleDateFormat;
+import java.util.List;
+import java.util.Properties;
+
+/**
+ * Created by pankaj on 10/21/16.
+ */
```

Comment:
please remove those - we follow [collective code ownership](http://www.extremeprogramming.org/rules/collective.html) and such created by don't help in that (feel free to remove any created by Konrad if there's any in there :-))